exercises/triangle/triangle.go

```diff
@@ -1,16 +1,29 @@
+// This is a "stub" file.  It's a little start on your solution.
+// It's not a complete solution though; you have to write some code.
+
+// Package twofer should have a package comment that summarizes what it's about.
+// https://golang.org/doc/effective_go.html#commentary
 package triangle
 
 const testVersion = 3
 
-func KindFromSides(a, b, c float64) Kind
-
 // Notice KindFromSides() returns this type. Pick a suitable data type.
 type Kind
 
-// Pick values for the following identifiers used by the test program.
-NaT // not a triangle
-Equ // equilateral
-Iso // isosceles
-Sca // scalene
+const (
+    // Pick values for the following identifiers used by the test program.
```

Comment:
Wonder if we should link somewhere in the hints about [const values and iota](https://splice.com/blog/iota-elegant-constants-golang/)?

Reply:
Good idea. I remember when I discovered this through Exercism, mind :boom: